src/main/java/com/launchdarkly/client/LDClient.java

```diff
@@ -344,7 +343,7 @@ private JsonElement evaluate(String featureKey, LDUser user, JsonElement default
     try {
       FeatureFlag featureFlag = config.featureStore.get(featureKey);
       if (featureFlag == null) {
-        logger.warn("Unknown feature flag " + featureKey + "; returning default value");
+        logger.debug("Unknown feature flag " + featureKey + "; returning default value");
```

Comment:
I agree with this change-- we've seen enough cases where an unknown FF is expected (e.g. test scenarios), so `debug` does feel appropriate to me.